.ci/ci_job_flags.sh

```diff
@@ -185,6 +185,13 @@ case "${CI_JOB}" in
 	export KATA_HYPERVISOR="firecracker"
 	export KUBERNETES="yes"
 	;;
+"FIRECRACKER_JAILER")
```

Comment:
can we put `"FIRECRAKER"|"FIRECRACKER_JAILER") `? I see that both CI jobs have the same configuration